test/integration/users_signup_test.rb

```diff
@@ -22,8 +27,36 @@ class UsersSignupTest < ActionDispatch::IntegrationTest
         confirm_password: "password" } }
     end
     follow_redirect!
-    assert_template 'users/show'
-    assert is_logged_in?
+    #assert_template 'users/show'
+    #assert is_logged_in?
```

Comment:
why comment these out?

Reply:
In CH11  on page 647 it was commented out, reason was the create method within the controller was modified to redirect to root url and flat a email message instead of logging in and redirecting to logged in user